specifyweb/frontend/js_src/lib/components/AppResources/EditorComponents.tsx

```diff
@@ -241,7 +241,7 @@ export function useCodeMirrorExtensions(
     const language =
       mode === 'json'
         ? [json(), jsonLinter(handleLinted)]
-        : mode === 'properties'
+        : mode === 'properties' || mode === 'txt'
```

Comment:
while it might fix the issue, it's not a correct fix

if you look at this, `txt` is used when code is unable to determine that the resource is a `properties` file:

https://github.com/specify/specify7/blob/8ce714a19c2099c9c675194a6f6eb9aa70dc7213/specifyweb/frontend/js_src/lib/components/AppResources/hooks.ts#L210-L211

thus, we should fix the logic so that it properly detects this file as properties
and if we can't do that, then not fix this at all, as properties files won't be needed once https://github.com/specify/specify7/issues/3169 is implemented (yet another reason to do it)

Reply:
see comment below